src/script/components/MessagesList/Message/MemberMessage.tsx

```diff
@@ -73,13 +73,6 @@ export const MemberMessage: React.FC<MemberMessageProps> = ({
   const isMemberLeave = message.isMemberLeave();
   const isMemberChange = message.isMemberChange();
 
-  const handleShowMoreClick = (event: React.MouseEvent<HTMLElement, MouseEvent>) => {
-    const clickedOnShowMore = (event.target as HTMLElement).closest('.message-header-show-more');
-    if (clickedOnShowMore) {
-      onClickParticipants(highlightedUsers);
-    }
-  };
```

Comment:
This is now a native React click handler (not more listening to clicks on the parent component)